src/data/polynomial/identities.lean

```diff
@@ -64,6 +66,11 @@ private lemma poly_binom_aux3 (f : polynomial R) (x y : R) : f.eval (x + y) =
   f.sum (λ e a, (a *(poly_binom_aux1 x y e a).val)*y^2) :=
 by { rw poly_binom_aux2, simp [left_distrib, sum_add, mul_assoc] }
 
+/--
+A polynomial `f` evaluated at `x + y` can be expressed as
+the evaluation of `f` at `x`, plus `y` times the (polynomial) derivative of `f` at `x`,
+plus some element `k : R` times `y^2`.
```

Comment:
My thinking was
1. symbolically it's no different than reading the type signature
2. in words I get to point out that it is the polynomial derivative being used, and get to use the keyword "some" to indicate a particular element of `R`.